Shoko.Server/API/v3/Models/Shoko/Episode.cs

```diff
@@ -70,9 +93,9 @@ public static AniDB GetAniDBInfo(AniDB_Episode ep)
             var titles = RepoFactory.AniDB_Episode_Title.GetByEpisodeID(ep.EpisodeID);
             return new AniDB
             {
-                ID = ep.EpisodeID,
-                Type = (EpisodeType) ep.EpisodeType,
-                EpisodeNumber = ep.EpisodeNumber,
+                ID = ep.EpisodeID.ToString(),
```

Comment:
Why are you toStringing a number ID?

Reply:
I'm `toString`ing a number, since not all (potential) providers _may_ be using integers as ids, and in Shokofin it's converted to a string and only used as a string value anyway.